test/sql.test

```diff
@@ -41,17 +41,27 @@ namespace eval ::qcode::test {
     test sql_sort-1.0 {sql_sort} -setup {
     } -body {
         sql_sort name email user_code 
-    } -cleanup {} -result {name,email,user_code}
+    } -cleanup {} -result {"name","email","user_code"}
```

Comment:
The existing code would generate:
`column_a DESC, NULLS, LAST` which would be a sql error (note the commas)